cost-analyzer/values.yaml

```diff
@@ -520,3 +520,15 @@ serviceAccount:
 #  productKey: # apply business or enterprise product license
 #    key: ""
 #    enabled: false
+  # alertConfigs:
+  #   enabled: true
+  #   globalAlertEmails:
+  #   # - recipient@example.com
+  #   # - additionalRecipient@example.com
+  #   alerts:
+  #   # Daily budget alert on kubecost namespace at threshold 100 <currency>
+  #   - type: budget # supported: budget, weeklyUpdate
+  #     threshold: 100
+  #     window: daily # supported: daily, weekly
+  #     aggregation: namespace # supported: namespace, cluster
+  #     value: kubecost # aggregation value to alert on. i.e., namespace <name> or cluster <name>
```

Comment:
can this be empty or do we support wildcards? i.e. what if I want to set a budget alert for any namespace above X threshold.

is `value` able to to be set for recurring updates? is it optional?

Reply:
Wildcard should be supported since the aggregation API supports it.

Reply:
1. can this be unset? i.e. what if I want to set a budget alert for `any` namespace above X threshold.

2. is value able to to be set for recurring updates? if so, is it optional?

Reply:
1. It should be triggered by `*`, or `<prefix>*`
```
alerts:
    - type: budget
      threshold: X
      window: daily
      aggregation: namespace
      value: *
```
I could add a parser in between that converts `any` to `*` for the aggregation query, so users can input `any`
 into the field if that's more explicit to the user.

2. Yes, value should be used in most cases, whether it's `any`, `*`, or an exact namespace/cluster name. If left empty, the only case I can think of is total cost across all clusters, although I think it'd be more clear to make it mandatory and implement that by adding support through `type: allClusters`

Reply:
1. sounds great! just supporting and documenting `'*'` should be fine. i just want to you to think through what that experience looks like later... 

2. hm, i'm not sure i follow/agree. @AjayTripathy any thoughts? i do not think we should use the term `weeklyUpdate` that seems too rigid.

Reply:
re: weeklyupdate-- yes, let's remove that and callit recurringUpdate with a window/frequency of 7d.